.swift-version

```diff
@@ -0,0 +1 @@
+4.2
```

Comment:
let's remove this file please? (swift-env right? fine to use it, but let's not commit to repos)

Reply:
I was surprised to see SwiftFormat did use this, it specifically posted a warning asking for it. I figured we should go with the minimum-supported version to ensure compatibility. I didn’t add it for swiftenv support, but you’re totally right that would collide with it.

I’ll update so we go [with the commandline flag](https://github.com/nicklockwood/SwiftFormat#swift-version) 🤙